src/Events/Controllers/AppController.cs

```diff
@@ -139,11 +139,11 @@ public async Task<ActionResult<string>> Post([FromBody] AppCloudEventRequestMode
                 return Problem(errorMessage, null, 400);
             }
 
-            var source = new List<string> { $"%{org}/{app}%" };
+            string resource = $"urn:altinn:resource:altinnapp.{org}.{app}";
```

Comment:
If changing to the resource URN format the correct format as used by the resourceregistry would be:
`urn:altinn:resource:app_{org}_{app}`

Reply:
We will keep the wrong resource for now to get this functionality in. Work on fixing the resource for both event registration, and existing subscriptions and events in database will be fixed as a separate task

Reply:
We have the prefix as a constant in AuthorizationConstants, maybe this can be used here as well?

```suggestion
string resource = string.Concat(AuthorizationConstants.AppResourcePrefix, org, '.', app);
```

Reply:
Done